### ISSUE TITLE
Is there ImageNet Pretrain weights for Res101-GroupNorm32-Alignpadding?

### ISSUE THREAD

**Issue by engineer1109** [tensorpack/tensorpack#833]
I have seen the weights for Res50-GroupNorm32-Alignpadding?
However, Res50 is low in accuracy.
Is there ImageNet Pretrain weights for Res101-GroupNorm32-Alignpadding?

**Comment by ppwwyyxx** [comment]
There is no.

**Comment by engineer1109** [comment]
@ppwwyyxx How could I train a Res101-GroupNorm32-Alignpadding model？

**Comment by ppwwyyxx** [comment]
You can modify the resnet training code following what's in the fasterrcnn basemodel.

**Comment by engineer1109** [comment]
@ppwwyyxx  Alignpadding means padding

**Comment by engineer1109** [comment]
@ppwwyyxx  Alignpadding means what?

**Comment by engineer1109** [comment]
@ppwwyyxx Have you tested the mAP on groupnorm?

**Comment by ppwwyyxx** [comment]
Everything I have are in the table.

**Comment by ppwwyyxx** [comment]
Training code for the GN model is now available at https://github.com/ppwwyyxx/GroupNorm-reproduce/
